tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
excellent, now it is making sense to me...  @gregw do you know where that link came from?

https://github.com/GoogleCloudPlatform/jetty-runtime/blob/master/jetty9/src/main/docker/Dockerfile#L41